Core/clim-core/gadgets.lisp

```diff
@@ -2886,62 +2895,94 @@ it in a layout between two panes that are to be resizeable.  E.g.:
 (defmethod compose-space ((gadget clim-extensions:box-adjuster-gadget)
                           &key width height)
   (declare (ignore width height))
-  (if (eq (orientation gadget) :vertical)
-      (make-space-requirement :min-width 6 :width 6 :max-width 6
-                              :min-height 1 :height 1)
-      (make-space-requirement :min-height 6 :height 6 :max-height 6
-                              :min-width 1 :width 1)))
+  (let ((major-size +box-adjuster-gadget-major-size+)
+        (minor-size +box-adjuster-gadget-minor-size+))
+    (if (eq (orientation gadget) :vertical)
```

Comment:
if it is either vertical or horizontal ecase would be more specific

Reply:
I thought so as well when reading the original code. Should have acted on the thought. I will update the commit accordingly.

In general I would like to make dealing with orientations less annoying. My current thinking is something like this:

```lisp
(with-orientation (orientation gadget)
  …
  (bounding-rectangle-major-size …)
  (bounding-rectangle-max-major …)
  …)
```

where `with-orientation` would bind a dynamic variable and "major" and "minor" accessors would read it.

Reply:
I think that "dada" evil macro did what you have suggested in a hacky way, please see "panes.lisp" and vbox/hbox panes.